fastai/vision/augment.py

```diff
@@ -517,7 +574,12 @@ def affine_mat(*ms):
                   stack([t0(ms[0]), t0(ms[0]), t1(ms[0])], dim=1)], dim=1)
 
 # Cell
-def flip_mat(x, p=0.5, draw=None, batch=False):
+def flip_mat(
+    x:Tensor, # The input Tensor
+    p=0.5, # Probability of appying transformation
+    draw:(int, [int], callable)=None, # User defined behavior of batch transformation
+    batch:bool=False # Same behavior for the whole batch if `True`
+):
```

Comment:
Similar to the above comment. Although I'm not sure about my doc string suggestion for draw is strict improvement. Perhaps @marii-moe has a better suggestion?
```suggestion
def flip_mat(
    x:Tensor, # The input Tensor
    p=0.5, # Probability of applying flip
    draw:(int, [int], callable)=None, # Custom flips instead of random
    batch:bool=False # Apply identical flip to entire batch
):
```

Reply:
I think that is fine. Callable would generally be defined as a random distribution, for example your could change it to use a log_uniform distribution, uniform distribution, or gaussian distribution. 

Though I still prefer how short your explanation is, and the flexibility it gets across so I agree with going with it.